silo/api.py

```diff
@@ -274,24 +274,25 @@ class SiloViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = SiloSerializer
     lookup_field = 'id'
     # this permission sets seems to break the default permissions set by the restframework
-    # permission_classes = (IsOwnerOrReadOnly,)
+    # permission_classes = (IsOwnerOrReazzdOnly,)
```

Comment:
Why is this in here if it is commented out. And why did you change it to `IsOwnerOrReazzdOnly` ??

Reply:
I didn't see this! hahaha